src/main/java/de/zalando/ep/zalenium/servlet/renderer/LiveNodeHtmlRenderer.java

```diff
@@ -55,9 +55,15 @@ public String renderSummary() {
         // Adding live preview
         int noVncPort = proxy.getRegistration().getNoVncPort();
         String noVncIpAddress = proxy.getRegistration().getIpAddress();
-        String noVncViewBaseUrl = "%s/vnc/host/%s/port/%s/?nginx=%s:%s&view_only=%s";
-        String noVncReadOnlyUrl = String.format(noVncViewBaseUrl, contextPath, noVncIpAddress, noVncPort, noVncIpAddress, noVncPort, "true");
-        String noVncInteractUrl = String.format(noVncViewBaseUrl, contextPath, noVncIpAddress, noVncPort, noVncIpAddress, noVncPort, "false");
+        String websockifyContextPath = null;
+        if (contextPath == null || contextPath.isEmpty()) {
```

Comment:
For this `if`, you could just use `Strings.isNullOrEmpty()`.

Or even leave the whole thing like:
```suggestion
        String websockifyContextPath = Strings.isNullOrEmpty(contextPath) ? "" : contextPath + "/";
```